Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/RideDetails.yaml

```diff
@@ -26,6 +26,8 @@ RideDetails:
     fleetOwnerId : Maybe Text
     defaultServiceTierName : Maybe Text
     createdAt : Maybe UTCTime
+    merchantId: Maybe (Id Merchant)
```

Comment:
These 2 line not required .. only removing from excluded field is enough.
If its kept for better user readability then it's okay